app/models/validations/shared_validations.rb

```diff
@@ -19,17 +19,19 @@ def validate_numeric_min_max(record)
       next unless record[question.id]
 
       field = question.check_answer_label || question.id
+      min = [question.prefix, question.min].join("")
```

Comment:
similarly - if we are prepending prefixes eg currency, do we want to append suffixes eg %?

Reply:
yup! good spot!